src/ds/d400/d400-active.cpp

```diff
@@ -34,7 +34,7 @@ namespace librealsense
         auto pid = this->_pid;
         auto& depth_ep = get_depth_sensor();
 
-        if( pid == ds::RS457_PID || pid == ds::RS430_GMSL_PID)
+        if( _is_mipi_device )
```

Comment:
Please make sure that every place you now use `_is_mipi_device` has it initialized.
I remember I could not use it everywhere as there were places in the code that were called before the `is_mipi_device` was set